github_issue_summarization/demo/README.md

```diff
@@ -94,14 +95,14 @@ export ENV=gh-demo-1003
 Set a bucket for the job output
 ```
 DAY=$(date +%Y%m%d)
-ks param set --env=${ENV} tfjob-v1alpha2 output_model_gcs_bucket kubecon-gh-demo
-ks param set --env=${ENV} tfjob-v1alpha2 output_model_gcs_path gh-demo/${DAY}/output
+ks param set --env=${ENV} tfjob output_model_gcs_bucket kubecon-gh-demo
```

Comment:
Did this work?
It looks like the command line arguments changed in a recent refactor of the code
https://github.com/kubeflow/examples/blob/37dd52f49d55ed92675282392caa10abc606b552/github_issue_summarization/notebooks/train.py#L115

The arguments output_model_gcs_bucket no longer exists.

I'm working on a test for TFJob and updating the TFJob spec as needed.

Reply:
Were you using a previously built docker image as opposed to an image built from the current version of the code?

I'd be OK with submitting as is and then fixing in a follow on PR. Once your PR is merged I can sync and make the requisite changes to tfjob.jsonnet to work with the latest version of the code.

Reply:
ah - yes, the codelab uses a pretty old image. That seems like a parameter that we want to put back. let's merge this first and then fix.

Reply:
The new version of the code supports gcs you just use a full GCS path for the input/output arguments.
The code uses the prefix "gs://" to detect that its a a GCS path and not a local filesystem path.